apps/native/app/src/screens/inbox/inbox.tsx

```diff
@@ -501,7 +548,13 @@ export const InboxScreen: NavigationFunctionComponent<{
                 onPress={onPressMarkAllAsRead}
               />
             </ListHeaderWrapper>
-            {opened || archived || bookmarked ? (
+            {opened ||
```

Comment:
~nit maybe it would be clearer to move all those checks into a descriptive boolean